test/e2e/tests/api-usage/account-tracker-api-usage.spec.ts

```diff
@@ -0,0 +1,173 @@
+  const allInfuraJsonRpcRequests: JsonRpcRequest[] = [];
+  let seenRequests;
+  let seenProviderRequests;
+
+  for (const m of mockedEndpoint) {
+    seenRequests = await m.getSeenRequests();
+    seenProviderRequests = seenRequests.filter((request) =>
+      request.url.match('infura'),
+    );
+
+    for (const r of seenProviderRequests) {
+      const json = await r.body.getJson();
+      if (json !== undefined) {
+        allInfuraJsonRpcRequests.push(json);
+      }
+    }
+  }
+
+  return allInfuraJsonRpcRequests;
+}
+
+describe('Account Tracker API Usage', function () {
+  it('should not make eth_call or eth_getBalance requests before the UI is opened and should make those requests after the UI is opened', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().withNetworkControllerOnMainnet().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test?.fullTitle(),
+        testSpecificMock: mockInfura,
+      },
+      async ({
+        driver,
+        mockedEndpoint,
+      }: {
+        driver: Driver;
+        mockedEndpoint: MockedEndpoint[];
+      }) => {
+        await driver.delay(3000);
+        let allInfuraJsonRpcRequests = await getAllInfuraJsonRpcRequests(
+          mockedEndpoint,
+        );
+        let ethCallAndGetBalanceRequests = allInfuraJsonRpcRequests.filter(
+          ({ method }) => method === 'eth_getBalance' || method === 'eth_call',
+        );
+
+        assert.ok(
+          ethCallAndGetBalanceRequests.length === 0,
+          'An eth_call or eth_getBalance request has been made to infura before opening the UI',
+        );
+
+        await unlockWallet(driver);
+        await driver.delay(3000);
+
+        allInfuraJsonRpcRequests = await getAllInfuraJsonRpcRequests(
+          mockedEndpoint,
+        );
+        ethCallAndGetBalanceRequests = allInfuraJsonRpcRequests.filter(
+          ({ method }) => method === 'eth_getBalance' || method === 'eth_call',
+        );
+
+        assert.ok(
+          ethCallAndGetBalanceRequests.length > 0,
+          'No eth_call or eth_getBalance request has been made to infura since opening the UI',
+        );
+      },
+    );
+  });
+
+  it('should not make eth_call or eth_getBalance requests after the UI is closed', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().withNetworkControllerOnMainnet().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test?.fullTitle(),
+        testSpecificMock: mockInfura,
+      },
+      async ({
+        driver,
+        mockedEndpoint,
+      }: {
+        driver: Driver;
+        mockedEndpoint: MockedEndpoint[];
+      }) => {
+        await unlockWallet(driver);
+        await driver.delay(3000);
+        const initialInfuraJsonRpcRequests = await getAllInfuraJsonRpcRequests(
+          mockedEndpoint,
+        );
+
+        await driver.openNewURL('chrome://extensions');
```

Comment:
It looks like this makes the firefox test fail. Could we selectively go to 

```
if (process.env.SELENIUM_BROWSER === 'chrome') {
  await driver.openNewURL('chrome://extensions');
}
if (process.env.SELENIUM_BROWSER === 'firefox') {
  await driver.openNewURL('about:debugging#/runtime/this-firefox');
}   
```

Reply:
I fixed it by replacing the url with `about:blank` https://github.com/MetaMask/metamask-extension/pull/26141/commits/a31544443deed3f068d5ec04857d22a6ca82fc0d

Reply:
oh nice :heart_eyes: ty!!